src/main/java/io/vertx/core/AbstractVerticle.java

```diff
@@ -100,8 +100,10 @@ public JsonObject config() {
    * and call the startFuture some time later when start up is complete.
    * @param startFuture  a future which should be called when verticle start-up is complete.
    * @throws Exception
+   * @deprecated Use {@link Verticle#start(Promise)}
```

Comment:
I think it should be `AbstractVerticle#start(Promise)` as link

Reply:
I mean this is clearly a message for people extending this interface and nobody is calling it (beside the vertx deployment), so the message rather should be _instead override {@link AbstractVerticle#start(Promise)}_

Reply:
Done

Reply:
the wording should be changed too to _instead override {@link AbstractVerticle#start(Promise)}_

Reply:
There you go